cluster-manifests/base/cluster-baseline-settings/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ns-cluster-baseline-settings.yaml
```

Comment:
Any thoughts on naming this just `ns.yaml` to match the one in `a0042` ?  Not critical to change, just a thought.

Reply:
sure thing! in the spirit of trying to minimize the conflicts, please let me take care of this as part of the next PR as this will have impact on the GitHub workflow.

Reply:
Done here to honor the feedback ❤️  (will take over the conflicts in next PR) | Addressed from [654ee](https://github.com/mspnp/aks-secure-baseline/pull/183/commits/654ee5b382868407b589bb0595a34fe9acd38825)